src/System.IO.Compression/src/System/IO/Compression/DeflateManaged/DeflateManagedStream.cs

```diff
@@ -14,38 +14,29 @@ internal sealed partial class DeflateManagedStream : Stream
     {
         internal const int DefaultBufferSize = 8192;
 
-        private Stream _stream;
+        private Stream? _stream;
         private bool _leaveOpen;
-        private InflaterManaged _inflater;
-        private byte[] _buffer;
+        private InflaterManaged _inflater = null!;
```

Comment:
Initialized in the constructor, shouldn't need initialization.

Reply:
right